hw1/02-palindrome.js

```diff
@@ -1,3 +1,28 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+let result = document.getElementById("result");
+let answer = document.createElement("p");
+answer.setAttribute("class", "output");
+result.append(answer);
+
+function handleInput(event) {
+  var value = event.target.value;
+  //hint found from: https://www.freecodecamp.org/news/two-ways-to-check-for-palindromes-in-javascript-64fea8191fd7/
+  var valueReversed = value.split("").reverse().join("");
+
+  if (isNaN(value)) {
+    answer.textContent = "Enter a number";
+    answer.style.color = "red";
+  } else if (value < 0) {
+    answer.textContent = "Enter a positive number";
+    answer.style.color = "red";
+  } else if (value === valueReversed) {
+    answer.textContent = "Yes. This is a palindrome!";
+    answer.style.color = "green";
+  } else {
+    answer.textContent = "No, Try again";
+    answer.style.color = "red";
+  }
+}
```

Comment:
The Airbnb styling guide says to use single quotes (' ') instead of double quotes (" ") for JavaScript. Another thing you could do since the homework asks to use Bootstrap classed for styling, instead of using `answer.style.color` to add CSS styling to the element you should add Bootstrap classes to the element instead. Everything looks good besides these two issues!